public/content/en/gems/ndslice.md

```diff
@@ -0,0 +1,120 @@
+# Multidimensional arrays
+
+D provides support for multidimensional arrays directly in its standard library
+Phobos. It allows fast iteration, access and manipulation of multidimensional
+data. In comparison to popular software packages in other languages (e.g. `ndarray`
+in NumPy or `Array` in Julia), it doesn't become slow for computations not
+covered by preexisting functionality in a bound C-library nor forces the user
+to escape into another language. Additionally, due to its compile-time introspection
+capabilities, D enables further optimizations for superior performance.
+
+## Slices
+
+A `Slice` is a view on a multidimensional view on usually an underlying raw,
```

Comment:
Maybe we can also say that "`ndslice` generalizes the 1 dimensional random-access range concept to N dimensions. Like its siblings from `std.range`, the `ndslice` package provides many functions that take ranges as parameters and return ranges which are new view on existing data".